devstats/devstats_cli.yaml

```diff
@@ -0,0 +1,57 @@
+# Run the devstats container forever.
+# This is useful for being able to run commands interactively
+# via kubectl exec.
+# e.g. initializing the database.
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: devstats-cli
+  namespace: devstats
+spec:
+  selector:
+    matchLabels:
+      app: devstats-cli
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: devstats-cli
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: ndevstats
+        image: gcr.io/kubeflow-ci/devstats:latest
+        command:
+          - tail
+          - -f
+          - /dev/null        
+        env:
+        # Host will follow statefulset conventions
+        - name: PG_HOST
+          value: postgre-0.ghadb.devstats.svc.cluster.local
+        #  Keep in sync with the values on postgress container
+        #  Also some values presumably can't be changed without recreating the DB.
+        - name: PG_DB
+          value: gha
+        - name: PG_PASS
+          value: password
+
```

Comment:
You probably need to refer logs database here somehow.
You need two Postgres databases minmally:
- Your project's database - can have any name
- Logs database named `devstats` containing single table `gha_logs`, see INSTALL_*.md files.

Reply:
I have two databases both stored in a single instance of postgre.